lib/payload/script-engine.vala

```diff
@@ -432,8 +432,8 @@ namespace Frida {
 			}
 
 			public void post (string message, Bytes? data) throws Error {
-				if (state != LOADED && state != DISPOSED)
-					throw new Error.INVALID_OPERATION ("Only loaded scripts may be posted to");
+				if (state != LOADED && state != DISPOSED && state != LOADING)
+					throw new Error.INVALID_OPERATION ("Only loading/loaded scripts may be posted to");
```

Comment:
Would be good to turn this into a switch statement, and order the cases as in the enum. Perhaps we should also simplify the error message, as it's already misleading (we allow it in `DISPOSED` state too). Something along the lines of: `Only scripts still alive may be posted to`